plugin.py

```diff
@@ -518,6 +518,15 @@ def __init__(self, view: sublime.View) -> None:
         super().__init__(view)
         self._stored_region = sublime.Region(-1, -1)
 
+    def is_valid_view(self) -> bool:
+        transient = False
+        sheet = self.view.sheet()
+        if sheet is not None:
```

Comment:
`is not None` is rather verbose for something that can be either an object or `None`. `if sheet:` would work just fine. :)

Reply:
Personally I prefer explicit `None` checks since they're faster than a defined bool magic method.

```py
In [4]: class A:
   ...:     def __init__(self):
   ...:         self.id = 10
   ...:     def __bool__(self):
   ...:         return self.id != 0
   ...:

In [5]: y = A()
In [6]: %timeit if y is not None: ...
16.4 ns ± 0.00648 ns per loop (mean ± std. dev. of 7 runs, 100000000 loops each)

In [7]: %timeit if y: ...
82.9 ns ± 0.114 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)
```

However, I went and checked and this is one of the few classes that don't define a magic `__bool__` method in the `sublime` package, go figure.